integrationtests/server-integration/pom.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-integrationtests-parent</artifactId>
+        <version>11.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <properties>
+        <resources.dir>${basedir}/src/test/resources</resources.dir>
+        <ispnserver.project.dir>${basedir}/../../server/runtime</ispnserver.project.dir>
+        <server.build.dist>${ispnserver.project.dir}/target/${infinispan.brand.prefix}-server-${infinispan.brand.version}</server.build.dist>
+        <ispnserver.dist>${basedir}/target/infinispan-server</ispnserver.dist>
+    </properties>
+
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+
+    <artifactId>infinispan-server-integration</artifactId>
+    <name>Server Integration</name>
+    <description>Server Integration description</description>
+
```

Comment:
We should have a `<modules>` section here with all of the child modules.

Reply:
@ryanemerson are you sure? if look into `integrationtests` or `core` which have child modules it don't have `modules` section, also if you look into `infinispan-parent` I added
```
<module>integrationtests/server-integration</module>
<module>integrationtests/server-integration/wildfly-modules</module>
```

Reply:
It's not strictly necessary, but without it it's not possible todo the following:

```
cd integrationtests/server-integration
mvn clean install # All child modules installed
```

In the parent pom.xml, it's then only necessary to add `<module>integrationtests</module>`.

That being said, I see that we don't currently do that for all of the other modules under `integrationtests`, so don't worry about it.

Reply:
@ryanemerson I just added the `modules` section as you suggested